hapi/hedera-protobufs/services/state/network.proto

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
FYI - This file got moved to another directory outside of `hedera-protobufs` after a lengthy discussion on this [PR](https://github.com/hashgraph/hedera-services/pull/16581). This should hopefully also negate the need to change `module-info` again :)